packages/icons/web/v2/card_unknown_32.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="32" height="24" viewBox="0 0 32 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I think this icon looks different [on Figma](https://www.figma.com/file/vnFVuPNlqF45rkw1u9toBC/SumUp-Iconography?node-id=4003%3A14057)

Maybe this is an older version? The last one looks very similar to the 16 size

Reply:
Yeah, the export was messed up. I'll update it.